keps/sig-auth/740-service-account-external-signing/README.md

```diff
@@ -590,6 +583,10 @@ Initial PRs:
 - kubernetes/kubernetes#73110
 - kubernetes/kubernetes#125177
 
+1.32: Alpha release
```

Comment:
Given the [increased beta requirements](https://groups.google.com/a/kubernetes.io/g/dev/c/thuIea5aYtw/m/YYcOY2dhEwAJ) I'd like to see the following bits resolved:

1. In `###### How can an operator determine if the feature is in use by workloads?` question there's a TODO about increasing audit log surface, any decision one way or the other? 
2. Missing answer in `###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?`.
3. Missing answer in `### Troubleshooting` section.
4. Missing answer in `###### What are other known failure modes?`

Reply:
updated